Detections/QualysVM/NewHighSeverityVulnDetectedAcrossMulitpleHosts.yaml

```diff
@@ -0,0 +1,23 @@
+id: 84cf1d59-f620-4fee-b569-68daf7008b7b
+name: New High Severity Vulnerability Detected Across Multiple Hosts
+description: |
+  'This creates an incident when a new high severity vulnerability is detected across multilple hosts'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CustomConnector
+    dataTypes: 
+      - QualysHostDetection_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+query: |
+
+  let timeframe = ago(1h);
+  QualysHostDetection_CL
+  | where TimeGenerated >= timeframe
+  | mv-expand todynamic(Detections_s)
+  | extend Status = tostring(Detections_s.Status), Vulnerability = tostring(Detections_s.Results), Severity = tostring(Detections_s.Severity)
+  | where Status =~ "New" and Severity == "5"
+  | summarize dcount(NetBios_s) by tostring(Detections_s.QID)
```

Comment:
Timestamp is required, so we should bring thru the min and max - summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated)

then | extend timestamp = StartTime

Reply:
corrections have been made and pushed.